lib/placeholders/imageCard/readMe.md

```diff
@@ -0,0 +1,38 @@
+## Navbar
```

Comment:
Please Fix title here and in all relevant MD files.

Reply:
977c4770419b2a5b4289c7e62a8812e8b16cf095